src/Component/Symbolizer/Editor/Editor.example.md

```diff
@@ -0,0 +1,44 @@
+This demonstrates the use of Editor.
```

Comment:
You might want to add backticks to the components name.

Reply:
Done in https://github.com/terrestris/geostyler/pull/603/commits/2033f201aa81bb0a8b1983e7553dfdd34b5b2af0